waspc/data/Generator/templates/server/src/routes/operations/_action.js

```diff
@@ -1,17 +1,22 @@
 {{={= =}=}}
+import { 
+  deserialize as superjsonDeserialize,
+  serialize as superjsonSerialize,
+} from 'superjson'
 import { handleRejection } from '../../utils.js'
-
 {=& operationImportStmt =}
 
 export default handleRejection(async (req, res) => {
-  const args = req.body || {}
+  const args = (req.body && superjsonDeserialize(req.body)) || {}
```

Comment:
TODO: I should probably be catching something here.

Reply:
Maybe write a helper function called `getRequestBody` or similar so it's more readable.